WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/ProductDiscountView.swift

```diff
@@ -0,0 +1,134 @@
+import SwiftUI
+import Yosemite
+
+struct ProductDiscountView: View {
+    private let imageURL: URL?
+    private let name: String
+    private let stockLabel: String
+    private let productRowViewModel: ProductRowViewModel
+
+    private let minusSign: String = NumberFormatter().minusSign
+
+    @Environment(\.presentationMode) var presentation
+
+    @ObservedObject private var discountViewModel: FeeOrDiscountLineDetailsViewModel
+
+    init(imageURL: URL?,
+         name: String,
+         stockLabel: String,
+         productRowViewModel: ProductRowViewModel,
+         discountViewModel: FeeOrDiscountLineDetailsViewModel) {
+        self.imageURL = imageURL
+        self.name = name
+        self.stockLabel = stockLabel
+        self.productRowViewModel = productRowViewModel
+        self.discountViewModel = discountViewModel
+    }
+
+    var body: some View {
+        NavigationView {
+            ScrollView {
+                HStack(alignment: .center, spacing: Layout.spacing) {
+                    ProductImageThumbnail(productImageURL: imageURL,
+                                          productImageSize: Layout.productImageSize,
+                                          scale: 1,
+                                          productImageCornerRadius: Layout.frameCornerRadius,
+                                          foregroundColor: Color(UIColor.listSmallIcon))
+                    VStack {
+                        Text(name)
+                        CollapsibleProductCardPriceSummary(viewModel: productRowViewModel)
+                    }
+                }
+                .padding()
+                .overlay {
+                    RoundedRectangle(cornerRadius: Layout.frameCornerRadius)
+                        .inset(by: 0.25)
+                        .stroke(Color(uiColor: .separator), lineWidth: Layout.borderLineWidth)
+                }
+                .cornerRadius(Layout.frameCornerRadius)
+                .padding()
+                VStack(alignment: .leading) {
+                    DiscountLineDetailsView(viewModel: discountViewModel)
+                    HStack {
+                        Image(systemName: "arrow.turn.down.right")
+                            .foregroundColor(.gray)
+                        Text(Localization.discountLabel)
+                            .foregroundColor(.gray)
```

Comment:
Text normally has a semantic colour – I think it would be `.secondary` here? Both work, even in dark mode, so it's not a big problem, but it's not idiomatic.

Reply:
Good point, this went under the radar. Fixed (along similar ones) here: 8810531bc04ad2ef6f05396e4dafa2989255f12b